sdk/webpubsub/Azure.ResourceManager.WebPubSub/api/Azure.ResourceManager.WebPubSub.net8.0.cs

```diff
@@ -308,18 +527,38 @@ namespace Azure.ResourceManager.WebPubSub.Models
         public static bool operator !=(Azure.ResourceManager.WebPubSub.Models.AclAction left, Azure.ResourceManager.WebPubSub.Models.AclAction right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class ApplicationFirewallSettings : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.WebPubSub.Models.ApplicationFirewallSettings>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.WebPubSub.Models.ApplicationFirewallSettings>
```

Comment:
could we add the webpubsub as a prefix to this type?

Reply:
Renamed to `WebPubSubApplicationFirewallSettings`